packages/api/src/routes/medical/internal-hie.ts

```diff
@@ -0,0 +1,39 @@
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import { Request, Response } from "express";
+import Router from "express-promise-router";
+import httpStatus from "http-status";
+import { z } from "zod";
+import { getHieOverview } from "../../command/medical/admin/hie-overview";
+import { getUUIDFrom } from "../schemas/uuid";
+import { asyncHandler } from "../util";
+
+dayjs.extend(duration);
+
+const router = Router();
+
+const debugLevelSchema = z.enum(["info", "success", "error"]).optional();
+
+/**
+ * GET /internal/hie/patient/overview
+ *
+ * Retrieves the overall status of a patient across HIEs.
+ *
+ * @param req.query.patientId - The patient's ID.
+ * @param req.query.debugLevel - The level of details to include in the overview:
```

Comment:
seems like an optional param from the implementation

Reply:
yep, the default value was marked as such, but added "optional" to be more clear